index.html

```diff
@@ -0,0 +1,304 @@
+        <li><a href=""><img src="./img/Vector5.png" alt="angel"></a></li>
+        <li><a href=""><img src="./img/twitter.png" alt="twitter"></a></li>
+        <li><a href=""><img src="./imG/M.png"alt="medium"></a></li>
+    </ul>
+  </div>
+
+  <!-- flesh down -->
+  <div class="flesh-down">
+    <img src="./img/Group 55.png" alt="flesh-down">
+    <img src="./img/Work.png" alt="work">
+  </div>
+
+  <!-- My recent work -->
+  <div class="My-Recent-Work">
+    <h3>MY RECENT WORK</h3>
+  </div>
+
+  <!-- line3 -->
+  <div class="line2">
+    <center><img class="line-image" src="./img/Vector4(1).png" alt=""></center>
+  </div>
+
+  <!--PLACE HOLDER IMAGE-->
+  <div class="place-holder-img">
+    <img src="./img/ImgPlaceholder.png" alt="place holder">
+  </div>
+
+  <!-- MUTLI POST STORIES -->
+  <div class="title-post">
+    <h2>Multi-Post Stories</h2>
+  </div>
+  <div class="paragraph2">
+    <p>A daily selection of privately <br>
+       personalized reads; no accounts <br>
+       or sign-ups required. has been the <br>
+       industry's standard dummy text <br>
+       ever since the 1500s, when an <br>
+       unknown printer took a standard <br>
+       dummy text.</p>
+  </div>
+
+  <!-- languajes de programacion -->
+  <ul class="programming-languages">
+    <li><a href="">CSS</a></li>
+    <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+    <li><a href="">html</a></li>
+    <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+    <li><a href="">bootstrap</a></li>
+    <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+    <li><a href="">Ruby</a></li> 
+  </ul>
+
+  <!-- SEE PROJECT BUTTON -->
+  <div class="see-project">
+    <img src="./img/Enable.png"alt="see project"></div>
+  </div>
+
+
+<!--PLACE HOLDER IMAGE2-->
+<div class="place-holder-img">
+<img src="./img/ImgPlaceholder.png" alt="place holder">
+</div>
+<!-- MUTLI POST STORIES -->
+<div class="title-post">
+<h2>Multi-Post Stories</h2>
+</div>
+<div class="paragraph2">
+<p>A daily selection of privately <br>
+ personalized reads; no accounts <br>
+ or sign-ups required. has been the <br>
+ industry's standard dummy text <br>
+ ever since the 1500s, when an <br>
+ unknown printer took a standard <br>
+ dummy text.</p>
+</div>
+
+<!-- languajes de programacion -->
+<ul class="programming-languages">
+<li><a href="">CSS</a></li>
+<li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+<li><a href="">html</a></li>
+<li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+<li><a href="">bootstrap</a></li>
+<li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+<li><a href="">Ruby</a></li> 
+</ul>
+
+
+<!-- SEE PROJECT BUTTON -->
+<div class="see-project">
+<img src="./img/Enable.png"alt="see project"></div>
+</div>
+
+<!--PLACE HOLDER IMAGE2-->
+<div class="place-holder-img">
+<img src="./img/ImgPlaceholder.png" alt="place holder">
+</div>
+
```

Comment:
- Just like the previous reviewers have suggested, please wrap the whole of your work section around a section` tag 🙏 . This will improve the accessibility of your site 😄 😄